src/components/course-enrollments/CourseEnrollments.jsx

```diff
@@ -19,7 +19,7 @@ import * as selectors from './data/selectors';
 import * as actions from './data/actions';
 
 export class CourseEnrollments extends Component {
-  static contextType = AppContext;
+  static contextType = AppContext; // eslint-disable-line react/sort-comp
```

Comment:
nit: To avoid this eslint error, you might do `CourseEnrollments.contextType = AppContext;` outside of the component.